packages/block-editor/src/components/list-view/index.js

```diff
@@ -71,11 +73,20 @@ function ListViewComponent(
 		isExpanded = false,
 		showAppender = false,
 		blockSettingsMenu: BlockSettingsMenu = BlockSettingsDropdown,
+		rootClientId,
 	},
 	ref
 ) {
+	// This can be removed once we no longer need to support the blocks prop.
+	if ( blocks ) {
+		deprecated( '`blocks` property in `ListViewComponent`', {
```

Comment:
Tiny nit: should this line refer to how the component is exported? I noticed that other `deprecated` calls refer to the deprecated component in that way. E.g. something along the lines of:

```suggestion
		deprecated( '`blocks` property in `wp.blockEditor.__experimentalListView`', {
```